File: Java/lab22/src/main/java/org/example/Administrator.java
package org.example;

public class Administrator extends WorkerTemplate{
    @Override
    void doWork() {
        System.out.println("I administer in an organization");
    }
    @Override
    void takeOver() {
        System.out.println("I have a break from 13:00 to 14:00");
    }
    @Override
    void continueWork() {
        System.out.println("I continue administer in an organization");
    }

}

File: Python/lab22/Administrator.py
from WorkerTemplate import WorkerTemplate


class Administrator(WorkerTemplate):
    def do_work(self):
        print("I administer in an organization")

    def take_over(self):
        print("I have a break from 13:00 to 14:00")

    def continue_work(self):
        print("I continue administering in an organization")
